packages/frontend-shared/src/components/ShikiHighlight.vue

```diff
@@ -73,29 +73,36 @@ shikiWrapperClasses computed property.
 </template>
 
 <script lang="ts">
-import type { Highlighter } from 'shiki'
-import { getHighlighter, setOnigasmWASM, setCDN } from 'shiki'
+import type { Highlighter, ILanguageRegistration } from 'shiki'
+import { getHighlighter, setOnigasmWASM } from 'shiki'
 import onigasm from 'onigasm/lib/onigasm.wasm?url'
+import shikiCyTheme from '../public/shiki/themes/cypress.theme.json'
+const langTmsRaw = import.meta.globEager('../public/shiki/languages/*.tmLanguage.json')
+
+const langTms: ILanguageRegistration[] = Object.values(langTmsRaw).map((grammar: any) => {
+  return {
+    grammar,
+    id: grammar.name,
+    scopeName: grammar.scopeName,
```

Comment:
Personal preference so up to you, but I'd much prefer to leave `grammar` as is and not remap its properties (`grammar.name` -> `id` and `grammar.scopeName` -> `scopeName`). When we rename/move existing properties, sometimes it increases the cognitive load for others in the codebase and in this case it doesn't seem to be necessary since the properties are not deeply buried

Reply:
Ahhhh I just opened the [article you linked](https://github.com/shikijs/shiki/blob/main/docs/languages.md) and now I see why you're doing this. It'd be nice to have a comment like

> Put in format shiki needs for language customization. See https://github.com/shikijs/shiki/blob/main/docs/languages.md

Reply:
Will do.